### ISSUE TITLE
Coil-head move without control 

### ISSUE THREAD

**Issue by JerryLzysss** [HolographicWings/LethalExpansion#96]
I play with my friends and we find that the coil-head run so quickly and it would not hurt anyone. howerver, we see it and still run everywhere. it would not happened in official map.
our custom map are dust2, telamon and etc.

**Comment by nightbreed813** [comment]
I have seen the same issue except for the host. The coil-head will run with great speed at the host and will not react to other players. Once the host is dead they will not harm the other players but will run through the map at top speed. Because they run so quickly there is no way for the host to survive.

**Comment by HolographicWings** [comment]
> I play with my friends and we find that the coil-head run so quickly and it would not hurt anyone. howerver, we see it and still run everywhere. it would not happened in official map.
> our custom map are dust2, telamon and etc.

For comparison I need to know if this still happening on wateridge
I still have too few clues on this issue

**Comment by nightbreed813** [comment]
Yeah this is still happening on Wateridge on normal maps coil-head act as expected.

**Comment by HolographicWings** [comment]
> Yeah this is still happening on Wateridge on normal maps coil-head act as expected.

Can you try the 1.3.15 version of the mod and tell me if it still happen ?

**Comment by nightbreed813** [comment]
Took so many runs and you weren't kidding about the other issues but yes on 1.3.15 the coil-head does react as expected.

**Comment by HolographicWings** [comment]
> Took so many runs and you weren't kidding about the other issues but yes on 1.3.15 the coil-head does react as expected.

I finally know what to focus to fix the issue
But it's tricky..

**Comment by HolographicWings** [comment]
I uploaded a 1.3.27 in the preview channel
could you try it and tell me the result ?
https://thunderstore.io/c/lethal-company/p/HolographicWings/LethalExpansion/

**Comment by nightbreed813** [comment]
It is still broken on the preview build

**Comment by nightbreed813** [comment]
After playing a few times even tho it only reacts to the host its speed is normal so it is possible to escape him now.

**Comment by HolographicWings** [comment]
It's too tricky to fix ;c and i don't understand Unity Netcode
When fixing it all entrances will have ID 0
I need help to fix this issue

**Comment by nightbreed813** [comment]
I only have basic programming skills and know nothing about unity otherwise I would help out. I thank you for all the hard work you have put into this mod. Hopefully someone will be able to offer some insight. I have no problem testing it out when you find a fix.

**Comment by HolographicWings** [comment]
Hello @nightbreed813 
Could you test the 1.3.28 ?
https://thunderstore.io/c/lethal-company/p/HolographicWings/LethalExpansion_Preview/

**Comment by nightbreed813** [comment]
When you say on Wateridge the sdk will need updated does that mean I'll have to wait to test that level? I tried it on Wateridge and the coil-head does not react to anyone but the host still.

**Comment by HolographicWings** [comment]
> When you say on Wateridge the sdk will need updated does that mean I'll have to wait to test that level? I tried it on Wateridge and the coil-head does not react to anyone but the host still.

on Wateridge from https://thunderstore.io/c/lethal-company/p/HolographicWings/LEDefaultModules/

**Comment by nightbreed813** [comment]
Oof I'm a bad tester. Just double checked and I was on the wrong version. I'll update and retest tomorrow, well later today and let you know.

**Comment by HolographicWings** [comment]
Okay ^^

**Comment by JerryLzysss** [comment]
hello everyone
I try the version 1.3.29(maybe 1.3.30) with my friend,the problem still exist. i do not know the reason if it's because my map i test, i try itin dust2 and poltegust. the coil-head still react as expect. maybe offer some map what you use.

**Comment by nightbreed813** [comment]
I tested it with the latest version and it does work as expected now.

**Comment by HolographicWings** [comment]
> hello everyone
> I try the version 1.3.29(maybe 1.3.30) with my friend,the problem still exist. i do not know the reason if it's because my map i test, i try itin dust2 and poltegust. the coil-head still react as expect. maybe offer some map what you use.

Because all maps have to be updated with the new entrances

**Comment by HolographicWings** [comment]
> I tested it with the latest version and it does work as expected now.

Nice to know !

**Comment by JerryLzysss** [comment]
it is good to see the issue solved,but there are still lots of map using the old entrances.luckily,the **Chirstmas villager** moon has solved the problem.

**Comment by HolographicWings** [comment]
Because christmas village is mine :P
Peoples have to update their entrances to fix the coilhead
